kvbc/test/sparse_merkle_storage/kv_blockchain_db_editor_test.cpp

```diff
@@ -615,7 +615,7 @@ TEST_F(DbEditorTests, get_value_latest) {
           out_,
           err_));
   ASSERT_TRUE(err_.str().empty());
-  ASSERT_EQ("{\n  \"value\": \"0x00000050\"\n}\n", out_.str());
+  ASSERT_EQ("{\n\t\"value\":\"0x00000050\",\n\t\"version\":\"5\",\n}\n", out_.str());
```

Comment:
Can we maybe use two spaces and not `\t`? The only reason I am suggesting it is that other commands do it that way.